### PR TITLE
Feature: Add login by credentials

### PR DESCRIPTION
**Pull request by arthurvenicio**

### CODE REVIEW

File: src/app/(auth-routes)/login/page.tsx [hunk 1]
```diff
@@ -22,8 +23,7 @@ const formSchema = yup.object().shape({
 
 export default function LoginPage() {
   const [showPassword, setShowPassword] = useState(false);
```
Comment (JLTSF) [inline src/app/(auth-routes)/login/page.tsx:25]: Where is this constant used?

File: src/app/api/users/[id]/settings/route.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      },
+      include: {
+        User: false
+      }
+    });
+
+    if (setting) throw new Error('Setting not found');
+
+    return NextResponse.json(setting, { status: 200 });
+  } catch (e: any) {
```
Comment (JLTSF) [inline src/app/api/users/[id]/settings/route.ts:23]: What kind of error can be expected?

File: src/utils/document.ts
Comment (JLTSF) [inline src/utils/document.ts:1]: I liked

**Review by JLTSF — CHANGES_REQUESTED** [review]
I left some questions in your code

File: src/app/(auth-routes)/login/page.tsx (hunk 1 above)
Comment (arthurvenicio) [reply src/app/(auth-routes)/login/page.tsx:25]: This const is used to validate login form.

File: src/app/api/users/[id]/settings/route.ts (hunk 2 above)
Comment (arthurvenicio) [reply src/app/api/users/[id]/settings/route.ts:23]: If don't find an user settings in database, but the validation have a problem.

Comment (arthurvenicio) [reply src/app/api/users/[id]/settings/route.ts:23]: now i'm fix this error, thanks!

**Review by JLTSF — APPROVED** [review]

**Pull request merged**